app/app.go

```diff
@@ -89,16 +89,16 @@ type TestConfig struct {
 	DisablePing bool
 	// PingCallback is called when a ping was completed to a peer.
 	PingCallback func(peer.ID)
-	// ParSigExFunc provides an in-memory partial signature exchange.
-	ParSigExFunc func() core.ParSigEx
+	// ParSigExchangeFunc provides an in-memory partial signature exchange.
```

Comment:
As per ticket, this should be `ShareSigExchange(Func)`

Reply:
Done